neuralop/models/codano.py

```diff
@@ -0,0 +1,455 @@
+        Example: For a input with 3 variables (channels) and output_variable_codimension=2, the output will have 6 channels (3 variables × 2 codimension).
+
+    lifting : bool
+        Indicates whether to use a lifting module. The lifting module projects each input variable (or channel) into a higher-dimensional space. Default is True.
+        Example: If lifting=True and hidden_variable_codimension=32, each input channel is lifted to 32 channels.
+
+    hidden_variable_codimension : int
+        The number of hidden channels corresponding to each input variable (or channel). Each input channel is independently lifted 
+        to `hidden_variable_codimension` channels by the lifting operation. Default is 32.
+
+    lifting_variable_codimension : int
+        The number of channels in the hidden layers of lifting/projection module. Default is 64.
+
+    use_positional_encoding : bool
+        Indicates whether to use variable-specific positional encoding. If True, a learnable positional encoding is concatenated 
+        to each variable before the lifting operation. Default is False.
+
+    positional_encoding_dim : int
+        The dimension (number of channels) of the positional encoding. Default is 8.
+
+    positional_encoding_modes : list
+        Number of Fourier modes used in positional encoding along each dimension. The positional embeddings are directly learned 
+        in Fourier space, and this parameter must be specified when `use_positional_encoding=True`. Default is None.
+        Example: For a 2D input, positional_encoding_modes could be [16, 16].
+
+    static_channel_dim : int
+        The number of channels for static information, such as boundary conditions in PDEs. These channels are concatenated with 
+        each variable before the lifting operation. When `static_channel_dim > 0`, additional information must be provided during 
+        the forward pass. Default is 0.
+
+    n_variables : int
+        The number of variables. This parameter is required when `use_positional_encoding=True` to determine the number of learnable positional embeddings. Default is None.
+
+    use_horizontal_skip_connection : bool
+        Indicates whether to use horizontal skip connections, similar to U-shaped architectures. Default is False.
+
+    horizontal_skips_map : dict
+        A mapping that specifies horizontal skip connections between layers. Default is None.
+        Example: For a 5-layer architecture, horizontal_skips_map={4: 0, 3: 1} creates skip connections from layer 0 to layer 4 and layer 1 to layer 3.
+
+    n_layers : int
+        The number of codomain attention layers. Default is 4.
+
+    n_modes : list
+        The number of Fourier modes to use in integral operations in the CoDA-NO block along each dimension. Default is None.
+        Example: For a 5-layer 2D CoDA-NO, n_modes=[[16, 16], [16, 16], [16, 16], [16, 16], [16, 16]].
+
+    scalings : list
```

Comment:
Could this be `scaling_factors`? How is this different than the `output_shape` parameters of the FNO? Behavior should be clearer